frontend/package.json

```diff
@@ -36,6 +37,7 @@
     "connect-redis": "^8.0.1",
     "cross-env": "^7.0.3",
     "effect": "^3.12.1",
+    "effect": "^3.12.1",
```

Comment:
I think this double dependency might have been added by my rebase conflict resolution. Oops. Please remove it.